build.py

```diff
@@ -1333,6 +1333,7 @@ def create_docker_build_script(script_name, container_install_dir,
 
         if target_platform() == 'windows':
             runargs += ['--memory', FLAGS.container_memory]
```

Comment:
There is another place in build.py where --memory is used.
Also, build.py has a --container-memory flag that is documented as setting the memory in the windows container. If we are not going to support that anymore then we need to remove the --container-memory flag.

Reply:
@deadeyegoodwin  your advise is to extend that [baseargs](https://github.com/triton-inference-server/server/blob/main/build.py#L1303) list with `--isolation=process` argument? 

Regarding `--container-memory` flag I'm not capable to confirm that we are going to remove it.

Reply:
I don't know what --isolation=process does. But if you are replacing --memory with it, then it seems like you would want to replace it everywhere. Right?
If after your changes we don't use --container-memory anymore, then we should remove it.

Reply:
In my understanding docker is inherits LXC container and manage the process by [namespaces](https://man7.org/linux/man-pages/man7/namespaces.7.html) and [chroot](https://man7.org/linux/man-pages/man2/chroot.2.html) etc. Similar to LXC some process may compete for a process within docker engine and try to send specific instructions, to get network, file system etc , by sharing same socket we probably may get a situation when we braking that rule and we start seeing the situation where process trying to ocupate same resource in "namespace" unless isolation is enabled. 
I added some documentation in description for current [PR#4226](https://github.com/triton-inference-server/server/pull/4226)

---

This change won't remove `--container-memory` it will extend that call and appear as:
```
docker run -w /workspace/build --name tritonserver_builder --memory 8g 
--isolation=process -v \\.\pipe\docker_engine:\\.\pipe\docker_engine tritonserver_buildbase 
powershell.exe -noexit -File ./cmake_build.ps1 
```

Reply:
I see, I misread the change, I thought you were replacing --memory with --isolation... but you are just adding it.